manifests/00-custom-resource-definition.yaml

```diff
@@ -116,29 +116,30 @@ spec:
                       TODO: Add other useful fields. apiVersion, kind, uid?'
                     type: string
                 type: object
+                x-kubernetes-map-type: atomic
```

Comment:
Interesting.  This change and others like it came from <https://github.com/openshift/api/pull/1203/commits/b630f11fab2674c803ae326956460896b5627505>, which adds `x-kubernetes-map-type: atomic` to fields that use some Kubernetes API types.